scripts/convertConfig.ts

```diff
@@ -302,9 +313,12 @@ function convertConfig(dataDir: string, verbose: boolean) {
             features,
             books,
             languageCode: writingSystem.attributes.getNamedItem('code')!.value,
-            languageName: writingSystem
-                .getElementsByTagName('display-names')[0]
-                .getElementsByTagName('form')[0].innerHTML,
+            languageName:
+                writingSystem.childElementCount > 0
+                    ? writingSystem
+                          .getElementsByTagName('display-names')[0]
+                          .getElementsByTagName('form')[0].innerHTML
+                    : '',
```

Comment:
```suggestion
            languageName: writingSystem
                .getElementsByTagName('display-names')[0]
                ?.getElementsByTagName('form')[0].innerHTML,
```
In JS, accessing an element of an array that does not exist returns `undefined` instead of throwing an error as it does in Java.
Using `?` is sufficient to use [optional chaining](https://www.typescriptlang.org/docs/handbook/release-notes/typescript-3-7.html) to set `languageName` to `undefined` if the `display-names` do not exist (making the array length 0 and `array[0]` undefined).

Reply:
Thanks. I will try that out.

Reply:
It turns out that we require languageCode to be defined for Proskomma, but languageName is not. I did use this syntax on languageName.